install/kubernetes/cilium/values.yaml

```diff
@@ -395,6 +395,12 @@ bgpControlPlane:
   # -- Enables the BGP control plane.
   enabled: false
 
+  # -- SecretsNamespace is the namespace which BGP support will retrieve secrets from.
+  secretsNamespace:
+    # -- Create secrets namespace for BGP secrets.
+    create: true
+    name: cilium-bgp-secrets
+
```

Comment:
[This is a generated file](https://github.com/cilium/cilium/blob/af4d7619ecada7650f489d92c55b9a91d6317a29/install/kubernetes/cilium/values.yaml#L1-L2), please report your change to `install/kubernetes/cilium/values.yaml.tmpl` and then run

```
make -C install/kubernetes cilium/values.yaml
```

Reply:
Oops, missed that. Fixed.